graphql-dgs/src/test/kotlin/com/netflix/graphql/dgs/WordFilterDirective.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.graphql.dgs
+
+import graphql.schema.DataFetcherFactories
+import graphql.schema.GraphQLFieldDefinition
+import graphql.schema.idl.SchemaDirectiveWiring
+import graphql.schema.idl.SchemaDirectiveWiringEnvironment
+import java.util.function.BiFunction
+
+/**
+ *  An `@DgsDirective` example for test purpose.
+ *
+ *  @author Hantsy Bai <hantsy@gmail.com>
+ */
+@DgsDirective
+class WordFilterDirective : SchemaDirectiveWiring {
```

Comment:
If you provide 2 examples, it probably makes more sense to add another type of directive (for example INPUT_FIELD_DEFINITION or ARGUMENT_DEFINITION) as an example.

Reply:
WordFilterDirective is an unnamed directive that will be applied globally.

Reply:
Is that something desirable? Genious question

Reply:
It is just a feature supported in GraphQL Java.